packages/docusaurus-plugin-content-docs/src/client/docsClientUtils.ts

```diff
@@ -59,17 +59,28 @@ export function getActivePlugin(
 export const getLatestVersion = (data: GlobalPluginData): GlobalVersion =>
   data.versions.find((version) => version.isLast)!;
 
+export function sortVersionsByPathDepth(
```

Comment:
This function is an implementation detail of `getActiveVersion` and probably doesn't make sense as a standalone feature.

I'd prefer to not export it, and to only test it through `getActiveVersion`

What matters overall is that we can reliably get the active version, no matter the version metadata ordering.